src/components/UserInfo.js

```diff
@@ -1,32 +1,83 @@
 import Popup from "./Popup";
 
 export default class UserInfo {
-  constructor({ titleSelector, occupationSelector }) {
+  constructor({ titleSelector, occupationSelector, avatarSelector }) {
     this._profileTitleElement = document.querySelector(titleSelector);
     this._profileDescriptionElement =
       document.querySelector(occupationSelector);
+    this._profileAvatarElement = document.querySelector(avatarSelector);
+
+    //this._submitButtonSelector = config.submitButtonSelector;
+    //this._handleSavingButton = handleSavingButton;
   }
 
+  // const imageUrl = document.getElementById("update-avatar-modal").src;
+
+  
+
   getUserInfo() {
     // Returns an object containing information about the user
 
     // const inputValues = {
     return {
       name: this._profileTitleElement.textContent,
       description: this._profileDescriptionElement.textContent,
+      avatar: this._profileAvatarElement.value,
     };
 
     // return inputValues;
   }
 
-  setUserInfo({ name, description }) {
+  setUserInfo({ name, description, avatar }) {
     // const { name, description } = this.getUserInfo(inputValues);
-    console.log("setUSER WORKING");
+
+     // const submitButton = document.querySelector(".modal__button");
+    // submitButton.innerHTML = "Saving...";
+
     this._profileTitleElement.textContent = name;
     this._profileDescriptionElement.textContent = description;
+    this._profileAvatarElement.value = avatar;
```

Comment:
`    this._profileAvatarElement.value = avatar;
`

`.value` doesn't exist in the avatar image. You can have only `src` there


```js

    this._profileAvatarElement.src = avatar;

```